data/Li_2018/metadata.yml

```diff
@@ -0,0 +1,500 @@
+- var_in: gsmax
+  unit_in: mol{H2O}/m2/s
+  trait_name: leaf_stomatal_conductance_per_area_at_Asat
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: replicates
+  methods: Gas exchange measurements ^p Gas exchange measurements were conducted on
+    recent, fully expanded leaves. During drought stress, the healthiest fully expanded
+    leaf on each individual plant was chosen. Leaf gas exchange was measured using
+    a portable infrared gas analyser (Model 6400XT, Li-Cor, Lincoln, NE, USA) equipped
+    with the red-blue LED light source (6400-02B) and external CO2 injector. Leaf
+    net carbon assimilation rate (An), stomatal conductance (gs, mol m-2 s-1), and
+    leaf transpiration rate (E, mmol m-2 s-1) were measured by placing the leaf into
+    a 2 x 3 cm cuvette supplied with saturating light (1,500 umol m-2 s-1) and ambient
+    CO2 concentration (420 umol mol-1). Parameters were logged when readings were
+    visually stable (total coefficient of variation < 1), which was typically achieved
+    within 5 min. During these measurements, cuvette temperature was maintained at
+    25 °C, flow rate was set at 500 ml min-1, and RH in the cuvette was maintained
+    between 60% and 70%. A leakage correction was applied on Li-Cor 6400XT data according
+    to the manual, assuming a constant ambient CO2 concentration (420 umol mol-1).
+    Immediately after each gas exchange measurement, leaves were excised and placed
+    in a zip-lock bag humidified with moist paper towel and transported to the lab
+    in an ice-filled container for Psi leaf measurements. Psi leaf was measured using a
+    Scholander-type pressure chamber (PMS Instruments, Corvalis, OR, USA).
+- var_in: WD
+  unit_in: g/cm3
+  trait_name: wood_density
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: replicates
+  methods: An approximately 4-cm-long segment was collected from the basal region
+    of the stem. Bark was carefully removed with a razor blade, and sapwood fresh
+    volume was determined using the water displacement method (Markesteijn et al.,
+    2011). Then all samples were oven dried at 70°C for 48 hr to constant mass and
+    then weighed. Specific leaf area (m2 kg-1) was calculated as the ratio of leaf
+    area to dry mass. WD was calculated as the ratio of dry mass to fresh volume (g
+    cm-3).
+- var_in: SLA
+  unit_in: m2/kg
+  trait_name: leaf_mass_per_area
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: replicates
+  methods: Following stem hydraulic measurements, plants were separated into leaves
+    and stems. Leaf samples were further divided according to leaf position relative
+    to the stem segment used for hydraulic measurement. A representative subsample
+    of fresh leaves was collected from four per plants per species, and leaf area
```

Comment:
I'm wondering if the replicates for SLA should be 4?

Reply:
I read it as 4 leaves per individual, per species and left replicates as the number of individuals used for the mean of the population. The way we've been using replicates across studies doesn't feel consistent to me so I'm not really sure. What do you think?

Reply:
I'm fine with that interpretation - I'd just read it as 4 plants per species, but I think it is four per plant - with stray "s". I was overall confused by this study, because I think often the "replicates" are points on a curve through which a single line is fit. So I couldn't decide if this is 1 measurement or 24 (or 17/18). But let's just leave it.

Reply:
> I think there are two tricky bits here - it is Asat, not Amax - I'm surprised they call it Amax. Also, if you look at Table 2, they define it as "Maximum carbon assimilation rate under well-watered conditions". So I think that is the mean value they provide.

I've changed replicates to 4 because "Four individuals of each species were assigned to a well-watered treatment and were watered daily over the experimental period."